action.yml

```diff
@@ -22,6 +22,9 @@ inputs:
   public:
     description: 'Create new repositories with public visibility'
     default: 'false'
+  skip-login:
```

Comment:
I'm torn on whether this should be called `skip-login-check` to more accurately describe what it does (since this action never does a login), or `skip-login` for consistency with `action-up`. Either way, it might be helpful to clarify what's being skipped in the description.

Reply:
I can update it to `skip-login-check`, that is a bit more explict. And will update description

Reply:
i think skip-login is clear enough, consistency ftw :)